packages/plugin-ext/src/plugin/terminal-ext.ts

```diff
@@ -79,7 +79,7 @@ export class TerminalExtImpl implements Terminal {
         this.termProcessId = this.proxy.$createTerminal(nameOrOptions);
     }
 
-    sendText(text: string, addNewLine?: boolean): void {
+    sendText(text: string, addNewLine: boolean = true): void {
```

Comment:
`addNewLine` argument was optional. Why Did you change it?

Reply:
to declare it as a default parameter which means it's still optional + it provides a default value